src/components/VolunteerLineItem/VolunteerLineItem.tsx

```diff
@@ -0,0 +1,32 @@
+import styles from "./VolunteerLineItem.module.css";
+import type { VolunteerLineItemProps } from "./VolunteerLineItem.definitions";
+
+const VolunteerLineItem = ({ volunteer }: VolunteerLineItemProps) => (
+  <div className={styles["line-item-container"]}>
+    <div className={styles["line-item-container-left"]}>
+      <div className={styles["line-item-name-icon"]}>
+        {!volunteer.matchedStudents?.length ? (
+          <div className={styles["line-item-unmatched-status-icon"]} />
+        ) : (
+          <div className={styles["line-item-matched-status-icon"]} />
+        )}
+      </div>
+      <div className={styles["line-item-name"]}>
+        {`${volunteer.firstName} ${volunteer.lastName}`}
+      </div>
+    </div>
+    <div className={styles["line-item-container-right"]}>
+      {!volunteer.matchedStudents?.length ? (
+        <button className={styles["line-item-match-button"]}>
+          <div className={styles["line-item-button-title"]}>Match</div>
+        </button>
+      ) : (
+        <button className={styles["line-item-match-button"]}>
+          <div className={styles["line-item-button-title"]}>Matched</div>
+        </button>
```

Comment:
Can you just conditionally render just the word "Match" and not repeat the tags?